cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds

```diff
@@ -441,25 +449,8 @@ var __AdyenHelper: Object = {
         };
 
         jsonObject['applicationInfo'] = __AdyenHelper.getApplicationInfo();
-
-        //Set recurring type
-        var recurringType = __AdyenHelper.getAdyenRecurringType();
-        if (__AdyenHelper.getAdyenRecurringPaymentsEnabled() && !empty(recurringType)) {
-
-            if (saveCreditCard) {
-                jsonObject['recurring'] = {
-                    'contract': recurringType
-                };
-            } else {
-                // if save credit card hasn't been selected or guest order
-                if (__AdyenHelper.getAdyenGuestRecurringEnabled()) {
-                    jsonObject['recurring'] = {
-                        'contract': 'RECURRING'
-                    };
-                }
-
-            }
-        }
+        jsonObject['enableRecurring'] = __AdyenHelper.getAdyenRecurringEnabled();
+        jsonObject['enableOneClick'] = __AdyenHelper.getAdyenOneClickEnabled();
```

Comment:
Any implications of guest shoppers tokenizing OneClick?

Reply:
If enableRecurring is set, then this will also tokenise for guest shoppers. 
I will remove the configuration for guestTokenisation then.